src/docs/deployment/linux.md

```diff
@@ -235,6 +242,37 @@ apps:
   when snaps are strictly confined. This Flutter snap needs
   access to the network.
 
+**DBus interface**
+
+: The dbus interface provides a way for snaps to 
+  communicate over DBus. The snap providing the DBus 
+  service declares a slot with the well-known DBus name 
+  and which bus it uses. Snaps wanting to communicate 
+  with the providing snap’s service declare a plug for 
+  the providing snap. Note that a snap declaration is 
+  needed for your snap to be delivered via the snap store 
+  and claim this well-known DBus name (simply upload the 
```

Comment:
Is this "well known" name actually DBus or something else?

Reply:
@sfshaza2 I can only assume from my very limited experience:
First about name:

It seems that it not necessary should be "well-known" from the beginning. In reality: 
- every "new" DBus name will be reviewed manually after first snap will be loaded to Snapstore.
- after DBus name have been at least once manually reviewed and approved - it going to be "well-known" and not requiring manual review when you uploading new snap to Snapstore and will be approved automatically.

About your question and  maybe I give you a wrong answer:

I'm not sure that the name is actually the DBus itself, it seems like it is a name of DBus service for snap app.
For me it looks very similar to AndroidManifest.xml beginning
```
<manifest xmlns:android="http://schemas.android.com/apk/res/android"
    package="org.foo.my_super_cool_app">
```
But again, I maybe wrong, because I have just user look for what's going on under the hood.